### ISSUE TITLE
User Story # 13 - Delete User Profile 

### ISSUE THREAD

**Issue by amylour** [amylour/FreeFido#20]
As a **user**, I can **click a delete button in my user profile setup** so that I **can delete all personal data, photos and bookings that are connected to me on the website**.

**Acceptance Criteria**
1. 'Delete' button is accessible in 'Profile Setup' page, but not clickable accidentally.
2.  User is asked via modal window to confirm account deletion.

**Tasks**

- [x] Create 'delete' button
- [ ] Create modal for deletion confirmation
- [ ] Create deletion of user's details - ensure CASCADE is connected to the correct items

**Comment by amylour** [comment]
- Bug # 3 -> Info at **commit 976b5ce** -> When creating the delete account function, I managed to delete profile pk without deleting the user id which left the id/pk mismatched and caused issues for any new users created. I accessed the python shell and tried to match the id/pk again and ensure that the next registered account would be successful. Unfortunately not so I flushed the db and re-created my superuser. Issue resolved and delete code function removed until function is better understood.
